diff -up libplacebo-v3.104.0-rc1/src/meson.build.omv~ libplacebo-v3.104.0-rc1/src/meson.build
--- libplacebo-v3.104.0-rc1/src/meson.build.omv~	2020-12-27 21:01:29.992036361 +0100
+++ libplacebo-v3.104.0-rc1/src/meson.build	2020-12-27 21:01:48.504277321 +0100
@@ -70,8 +70,8 @@ else
   glslang_deps = [
     cxx.find_library('glslang',     required: glslang_req),
     cxx.find_library('HLSL',        required: glslang_req),
-    cxx.find_library('OGLCompiler', required: glslang_req),
-    cxx.find_library('OSDependent', required: glslang_req),
+    #    cxx.find_library('OGLCompiler', required: glslang_req),
+    #    cxx.find_library('OSDependent', required: glslang_req),
     cxx.find_library('SPIRV',       required: glslang_req),
     cxx.find_library('SPVRemapper', required: glslang_req),
   ]
